lib/stat_tracker.rb

```diff
@@ -22,4 +22,29 @@ def self.read_from_file(file)
     file_open.read
   end
 
+# ************* LeagueStatistics *************
+
+  def match_data_with_header(file)
+    self.instance_variable_get(file).transpose
+  end
+
+  def match_data_by_spec(file, column)
+    self.instance_variable_get(file).shift
+    self.instance_variable_get(file).map do |line|
+       line.unshift(line.delete(line[column]))
+    end
+  end
```

Comment:
I'm again curious about the `self.instance_variable_get` when we can just pass the instance variable `teams`, `games`, etc.